### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by lexavey** [projectdiscovery/nuclei#4231]
### Nuclei version: 
2.9.15

### Current Behavior:

```
└─$ go build
# github.com/projectdiscovery/utils/permission
../../../../../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block
        ../../../../../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot
../../../../../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block
        ../../../../../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw
```

### Expected Behavior:

Success build
### Steps To Reproduce:

```
git clone https://github.com/projectdiscovery/nuclei.git; \
cd nuclei/v2/cmd/nuclei; \
go build; 
```

### Anything else:

```
┌──(elliot@localhost)-[~/.../cmd/nuclei]
└─$ uname -a
Linux localhost 4.19.157-perf-g5cadb1b43998 #1 SMP PREEMPT Fri Jul 21 12:18:09 UTC 2023 armv8l Android
┌──(elliot@localhost)-[~/.../cmd/nuclei]
└─$ id
uid=10260(u0_a260) gid=10260(u0_a260) groups=10260(u0_a260),1079(ext_obb_rw),3003(inet),9997(everybody),20260(u0_a260_cache),50260(all_a260),99909997
┌──(elliot@localhost)-[~/.../cmd/nuclei]
└─$ 
```

**Comment by Mzack9999** [comment]
@lexavey Thanks for opening the issue - This will be fixed in https://github.com/projectdiscovery/utils/pull/268

**Comment by ehsandeep** [comment]
@lexavey this is now fixed and changes will be reflected in next nuclei release.

**Comment by lexavey** [comment]
> @lexavey Thanks for opening the issue - This will be fixed in [projectdiscovery/utils#268](https://github.com/projectdiscovery/utils/pull/268)

Thankyou <3
